### PR TITLE
kops: Migrate kops-aws-updown job to scenario

### PR DESCRIPTION
**Pull request by zmerlynn**
This is the safer job to migrate first (it only burns the `kops` repo if it fails, not the k8s repo).

### CODE REVIEW

File: scenarios/kubernetes_kops_aws.py [hunk 1]
```diff
@@ -200,7 +200,8 @@ def main(args):
 
     # Assume we're upping, testing, and downing a cluster by default
     PARSER.add_argument(
-        '--cluster', default='e2e-kops-aws-canary.test-aws.k8s.io',
+        '--cluster',
+        default=os.environ.get('KOPS_E2E_CLUSTER_NAME'),
```
Comment (krzyzacy) [inline scenarios/kubernetes_kops_aws.py]: you want to pass it down to --kops-cluster right?

`os.environ.get('KOPS_E2E_CLUSTER_NAME')` will be None

Comment (zmerlynn) [inline scenarios/kubernetes_kops_aws.py]: Yes, my bad. The order is off here. Hm.

Comment (krzyzacy) [inline scenarios/kubernetes_kops_aws.py]: set it in config.json might be better

Comment (zmerlynn) [inline scenarios/kubernetes_kops_aws.py]: Any objection to shoving this into `config.json` rather than the `.env` file?

Comment (zmerlynn) [inline scenarios/kubernetes_kops_aws.py]: Lol, racing comments.

File: jobs/ci-kubernetes-e2e-kops-aws-canary.env [hunk 2]
```diff
@@ -9,8 +9,8 @@
 # See https://github.com/kubernetes/kops/issues/775 for why NodePort is disabled
 GINKGO_TEST_ARGS=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[HPA\]|Dashboard|Services.*functioning.*NodePort
 
+KOPS_E2E_CLUSTER_NAME=e2e-kops-aws-canary.test-aws.k8s.io
```
Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-kops-aws-canary.env]: kill it here :-)

File: scenarios/kubernetes_kops_aws.py (hunk 1 above)
Comment (krzyzacy) [inline scenarios/kubernetes_kops_aws.py]: :smiling_imp:

File: jobs/ci-kubernetes-e2e-kops-aws-canary.env (hunk 2 above)
Comment (zmerlynn) [inline jobs/ci-kubernetes-e2e-kops-aws-canary.env]: Done.

**Comment by zmerlynn** [comment]
I'm lost on what the test is complaining about. It doesn't seem correct.

**Comment by krzyzacy** [comment]
aha you hit another WIP-hole

add an entry to https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap_test.py#L1297

**Comment by zmerlynn** [comment]
yeah, I just stumbled on that, too. Thanks!

**Comment by zmerlynn** [comment]
Looks like we're good to go, finally. PTAL!

File: scenarios/kubernetes_kops_aws.py [hunk 3]
```diff
@@ -200,8 +200,7 @@ def main(args):
 
     # Assume we're upping, testing, and downing a cluster by default
     PARSER.add_argument(
-        '--cluster', default='e2e-kops-aws-canary.test-aws.k8s.io',
-        help='Name of the aws cluster')
+        '--cluster', help='Name of the aws cluster (required)')
```
Comment (krzyzacy) [inline scenarios/kubernetes_kops_aws.py]: raise an exception when cluster is not set?

Comment (krzyzacy) [inline scenarios/kubernetes_kops_aws.py]: People would ignore the bold `(required)`!

**Comment by zmerlynn** [comment]
Done.

File: scenarios/kubernetes_kops_aws.py (hunk 3 above)
Comment (zmerlynn) [inline scenarios/kubernetes_kops_aws.py]: Done, PTAL

**Pull request merged**